.env.example

```diff
@@ -1,43 +1,151 @@
 #################################
-#   Environment configuration 
+#   Telegram Bot
 #################################
+#   This is important too, you need to create a bot that would do the notifications to your channel
```

Comment:
I think rearranging the order of tis made it a bit weird

Some of these texts are referenced here: https://docs.gnosis.io/protocol/docs/tutorial-telegram-bot (and reviewed by comms), maybe we can do a MIX.


Also, I think we should include a link to the tutorial and make it supervisible in the README and the env.example

Reply:
Check:
- https://github.com/gnosis/dex-telegram/pull/208/commits/3120500866ef6940a923f8479cf9924b6924cabf
- https://github.com/gnosis/dex-telegram/pull/208/commits/7165ef8eeeaac3d0cc76181ae7eec06112349580

Reply:
Cool! What about add it on the very top of the `.env.example` too?

Reply:
https://github.com/gnosis/dex-telegram/pull/208/commits/93c4bffd1981812b91a2cba1296a19f6fead3385